content/post/against-ab-tests.md

```diff
@@ -0,0 +1,93 @@
+---
+author: Michael Kaminsky
+title: "Against A/B Tests"
+draft: True
+publishDate: "2018-12-01"
+tags:
+ - Statistics
+ - Testing
+ - Web Analytics
+---
+
+The notion of an A/B test is premised on the fundamentally flawed assumption that there is one version of some treatment that is better for all users at all times. Analytics practitioners should embrace this variation and start designing systems that allow for (and encourage) non-binary outcomes of tests.
```

Comment:
Isn't the assumption that there is one version of some treatment that is better on average for most users?

Reply:
Yes. Good catch!

Reply:
Right. All users. Which is a big part of the problem.